biopet.wdl

```diff
@@ -60,6 +60,7 @@ task SampleConfig {
     String? preCommand
     String tool_jar
     Array[File]+ inputFiles
+    String stdoutFile
```

Comment:
This is not necessary. Wdl does specify a stdout() function.
https://github.com/openwdl/wdl/blob/master/versions/draft-2/SPEC.md#file-stdout